waypoints:
  pre_table1:
  - {x: 4.905, y: -29.033, theta: 1.872, frame: "map"}
  table1:
  - {x: 4.881, y: -28.146, theta: 1.93, frame: "map"}
  post_table1:
  - {x: 5.120, y: -28.681, theta: -1.34, frame: "map"}
  pre_table2:
  - {x: 4.905, y: -29.033, theta: 1.872, frame: "map"}
  table2:
  - {x: 4.881, y: -28.146, theta: 1.93, frame: "map"}
  post_table2:
  - {x: 5.120, y: -28.681, theta: -1.34, frame: "map"}

object_constraints:
  cube:
    bounds: { xmin: 0.034, xmax: 0.12, ymin: 0.034, ymax: 0.12, zmin: 0.034, zmax: 0.12 }
    location: { xmin: -.inf, xmax: .inf, ymin: -.inf, ymax: .inf, zmin: 0.5, zmax: .inf }

arm_poses:
  tuck: [1.32, 1.47, -0.20, 1.65, 0.00, 1.66, 0.00]
  ready: [1.55, -0.91, 0.98, -1.18, 0.14, -1.06, 0.0]
  stow: [1.32, -0.46, 2.76, 1.19, 0.00, 0.82, 0.00]
  drop: [0.54, 0.06, 1.31, -1.15, 1.61, -1.68, 0.0]

trajectories:
  tuck_to_stow:
  - [1.32, 1.47, -0.20, 1.65, 0.00, 1.66, 0.00]
  - [1.32, 1.47, -1.58, 1.65, 0.00, 1.66, 0.00]
  - [1.32, -0.46, -1.58, 1.65, 0.00, 1.66, 0.00]
  - [1.32, -0.46, -1.58, 1.65, 0.00, 0.82, 0.00]
  - [1.32, -0.46, 2.76, 1.19, 0.00, 0.82, 0.00]
  stow_to_tuck:
  - [1.32, -0.46, 2.76, 1.19, 0.00, 0.82, 0.00]
  - [1.32, -0.46, -1.58, 1.65, 0.00, 0.82, 0.00]
  - [1.32, -0.46, -1.58, 1.65, 0.00, 1.66, 0.00]
  - [1.32, 1.47, -1.58, 1.65, 0.00, 1.66, 0.00]
  - [1.32, 1.47, -0.20, 1.65, 0.00, 1.66, 0.00]
